docs/src/content/docs/ref/stdlib-deploy.mdx

```diff
@@ -11,42 +13,95 @@ To use this library, import `@stdlib/deploy`:
 import "@stdlib/deploy";
 ```
 
-## Messages
+## Functions
 
-### Deploy
+### cashback
+
+<Badge text="Gas-expensive" title="Uses 500 gas units or more" variant="danger" size="medium"/>
+<Badge text="Available since Tact 1.6.1" variant="tip" size="medium"/><p/>
 
 ```tact
-message Deploy {
-    queryId: Int as uint64;
-}
+fun cashback(to: Address);
 ```
 
-### DeployOk
+[Queues](/book/send#outbound-message-processing) an empty message to be sent with the [`SendRemainingValue{:tact}`](/book/message-mode#base-modes) mode to the destination [`Address{:tact}`](/book/types#primitive-types) `to`. This is the most gas-efficient way to send the remaining value from the incoming message to the given address.
+
+Attempts to queue more than $255$ messages throw an exception with [exit code 33](/book/exit-codes#33): `Action list is too long`.
+
+Usage example:
 
 ```tact
-message DeployOk {
-    queryId: Int as uint64;
+// Forward the remaining value back to the sender
+cashback(sender());
+```
+
+## Traits
+
+### EmptyDeployable
+
+<Badge text="Available since Tact 1.6.1" variant="tip" size="medium"/><p/>
+
+The trait `EmptyDeployable{:tact}` provides a simple unified mechanism for deployments by adding an empty `receive()` function, which handles all incoming messages with an empty body, i.e., `null{:tact}` body.
+
+To deploy a contract with this trait, send a message with [initial code and data](/book/expressions#initof) and an empty message body.
+
+Source code:
+
+```tact
+trait EmptyDeployable {
+    /// `null` message body receiver for deployment purposes.
+    receive() { }
```

Comment:
This is a dangerous receiver that does not refund: it should be at least obvious from the name.

What if someone sends a million TONs into such a contract?